composer.json

```diff
@@ -38,16 +38,15 @@
         }
     ],
     "require": {
-        "php": "^7.1",
+        "php": ">= 7.1.3",
```

Comment:
Can you remove the space between `>=` and `7.1.3`? (And also down at `"< 2.0"`)

Reply:
Hello,

Are you sure ? This is how it should be written according to [composer normalize](https://packagist.org/packages/ergebnis/composer-normalize).
```
~/d/g/phpspec-code-coverage > update-composer-json > composer normalize
./composer.json is already normalized.
 
```

Reply:
I never heard of composer normalize before. It seems it just ignores those whitespaces. I always use the official documentation: https://getcomposer.org/doc/articles/versions.md#stability-constraints

Although composer works just fine when executing ` composer require "php:>= 7.2"`, composer will not use a space when executing `composer require php` (results in ` "php": "^7.2"`).

I don't want to be nitpicky, and since this works just fine, it's ok for me.

Reply:
> Although composer works just fine when executing ` composer require "php:>= 7.2"`, composer will not use a space when executing `composer require php` (results in ` "php": "^7.2"`).

This changes is mostly to follow the trend in Symfony. I was already doing that in my packages since a long time and Symfony adopted it recently: https://github.com/symfony/symfony/pull/36876

Now, regarding adding a space or not... I obviously agree that there is no technical difference, but we will have to decide on one way to write things in `composer.json` one day, for consistency. This is why `composer normalizer` was created.

Closing the thread, feel free to re-open if needed.

Reply:
This wasn't about `>=` or `^`. This was weather there is a space in between:

* `>=7.1`
* `>= 7.1`

Reply:
Hello guys,

I just took a look at `composer normalize` but can't find a place in the examples where this space is added. Since the `composer require` command is mostly ran inside a CLI this space can cause trouble and need escaping.

I follow @jaylinski, we don't need to talk too much about space details but it seems strange. Isn't the `composer normalize` command running successfully even when you drop that space in the JSON ?

Reply:
Hello!

No, it complains if the space is not there.

Reply:
Ok, so let's close the discussion and accept those changes ^^. Thank you !